packages/fxa-settings/src/pages/Signup/ConfirmSignupCode/en.ftl

```diff
@@ -13,7 +13,7 @@ confirm-signup-code-heading = Enter confirmation code <span>for your { -product-
 confirm-signup-code-instruction = Enter the code that was sent to { $email } within 5 minutes.
 confirm-signup-code-input-label = Enter 6-digit code
 # Form button to confirm if the confirmation code entered by the user is valid
-confirm-signup-code-confirm-button = Confirm
+confirm-signup-code-submit-button = Confirm
```

Comment:
This ID change doesn't seem necessary. Was the intention was to also change the text from `Confirm` to `Submit`? 

I also can't find any changes in this PR associated with `confirm-signup-code-submit-button` apart from this ID change in the FTL file.

Reply:
@bcolsson I changed this because I noticed [we reference `confirm-signup-code-submit-button`](https://github.com/mozilla/fxa/blob/856dc61405a8588ed8eb370f285fb78464def696/packages/fxa-settings/src/pages/Signup/ConfirmSignupCode/index.tsx#L84-L85), but don't have any references to `confirm-signup-code-confirm-button` in the codebase currently. I believe this was a mistake from a previous PR.

Reply:
@LZoog - we chatted already but my recommendation would be to update the reference in the TSX file rather than the Fluent ID

Reply:
@bcolsson Change pushed! Thanks for catching.